conf/test.config

```diff
@@ -8,6 +8,9 @@
  */
 
 params {
+  config_profile_name = 'Test profile'
```

Comment:
Here in the `test.config`, all the links will be broken, now GitHub only supports `raw.githubusercontant.com`-based links. Could you update them? E.g.
https://raw.githubusercontent.com/nf-core/test-datasets/epitopeprediction/testdata/variants/variants.vcf

Reply:
Thanks @ggabernet. Just to be sure, that affects all (test) config files in `conf/` , right?

Reply:
yes, that affects them all

Reply:
Looks like it's broken again... :(

Reply:
Ok, I triggered the tests again and now all of them passed. 🎉